tests/electron/package.json

```diff
@@ -8,6 +8,7 @@
     "test": "electron --js-flags=\"--expose-gc\" --no-sandbox --enable-logging ./app/main.js"
   },
   "devDependencies": {
+    "cmake-js": "^6.1.0",
```

Comment:
Is this really needed? I hope we don't need users to install "cmake-js" to consume the package?

Reply:
This is for our tests only. Users will not need to do this

Reply:
I need to spent more time understanding the need for this, it smells like an undocumented peer dependency to me.